src/components/NftCollectionInventory/index.tsx

```diff
@@ -10,31 +9,18 @@ import styles from './styles.module.scss'
 import { getPrimaryColor } from '../../constants/common'
 import { explorerService } from '../../services/ExplorerService'
 import { handleNftImgError, patchMibaoImg } from '../../utils/util'
+import type { NFTItem } from '../../services/ExplorerService/fetcher'
 
 const primaryColor = getPrimaryColor()
 
-type NftCollectionInventoryItem = {
-  icon_url: string | null
-  id: number
-  token_id: string
-  owner?: string
-  standard: string
-  cell: {
-    cell_index: number
-    data: string
-    status: string
-    tx_hash: string
-  } | null
-}
-
 const NftCollectionInventory: React.FC<{
-  list: Array<NftCollectionInventoryItem>
+  list: Array<NFTItem>
```

Comment:
How about using `NFTItem[]` instead of `Array<NFTItem>`?

Reply:
Agreed, a rule should be added later to restrict the use of `[]` as much as possible instead of `Array`.

Reply:
done